gradle/gitLatestTag.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+git tag | grep -E '^[0-9]' | sort -V | tail -1
```

Comment:
You shouldn't need a bash script for this.

https://github.com/JuulLabs/juul-app-android/blob/05589f86de2aad7453768d15a4e415fabc65f367/gradle/versions.gradle

Reply:
Yeah this was a hack as using `ProcessBuilder` does not allow for multiple piped commands so easily. I'll change it to use a sub shell and `project.exec` and delete this.